scripts/prepare_test_env.sh

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2018-2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#!/bin/bash
+
+SUBJECT=ca-ing
+export DOMAIN_NAME=$1
+export PROXY=$2
+export DEX_DOMAIN_NAME="dex.$DOMAIN_NAME"
+export MGMT_DOMAIN_NAME="mgt.$DOMAIN_NAME"
+echo "Fetching CA for $MGMT_DOMAIN_NAME"
+./get_cert.sh $MGMT_DOMAIN_NAME $SUBJECT $PROXY > ca.pem
+cat ./ca.pem
+export REQUESTS_CA_BUNDLE=`pwd`/ca.pem
```

Comment:
in readme we have `export REQUESTS_CA_BUNDLE=/etc/ssl/certs/`  which is different then this.

Reply:
in readme we recommended cert installation - here we just point to specific cert

Reply:
so why we do not follow readme recomendation here?

Reply:
ok, that's actually better solution, so I'll add certs installation